### PR TITLE
images: use smaller optimized files to reduce pdf file size

### PR DESCRIPTION
**Pull request by entorb**

### CODE REVIEW

File: layout/hp-header.tex [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 \usepackage[bookmarks=true,unicode=true,pdfborder={0 0 0},
 	pdftitle={Harry Potter and the Methods of Rationality},
-	pdfauthor={Eliezer Yudkowsky}, breaklinks={true},
+	pdfauthor={Yudkowsky, Eliezer}, breaklinks={true},
```
Comment (rrthomas) [inline layout/hp-header.tex:16]: What motivated this change? It seems to be more common with a single author to use the names in normal written order.

Comment (entorb) [reply layout/hp-header.tex:16]: In calibri (ebook Library tool) it seams to be standard to have "lastname, firstname", as this is used for sorting the books. Hence I suggest to use this order in the pdf as well

Comment (rrthomas) [reply layout/hp-header.tex]: I did some searching, and I can find nothing about what is expected. I had look at a couple of files, in Calibre and with `pdfinfo`, and in each case the author name was shown in "natural" order by `pdfinfo` (and in one case, I was able to confirm this by looking at RDF in the file; in the other, I'm not sure how calibre gets the information for the other!). If I look at right click→Show Book details… in calibre, it has another field "Author sort" which has the author in "Last name, First name" order.

In short, it seems as though Calibre is quite capable of parsing names. I guess there are cases where it would be necessary to use "Last Name, First Name", e.g. "von Bigwig, Jack", but this does not seem to be one of those; and we will not have the problem of naive programs displaying the name in an unnatural order to the reader if we stick with things as they are.

I do not regard my findings as definitive, so if you have any further information, do share it!

Comment (entorb) [reply layout/hp-header.tex]: As you prefer, I just felt like writing it the other way round, but have no strong preference, so keeping it as it is would also be fine with we ;-)
